ajax/libs/perliedman-leaflet-control-geocoder/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "perliedman-leaflet-control-geocoder",
+  "version": "1.5.1",
+  "description": "Extendable geocoder with builtin support for Nominatim, Bing, Google, Mapbox, Photon, What3Words, MapQuest, Mapzen, HERE",
+  "filename": "Control.Geocoder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/perliedman/leaflet-control-geocoder.git"
+  },
+  "keywords": [
+    "leaflet",
+    "geocoder",
+    "locations",
+    "nominatim",
+    "bing",
+    "google",
+    "mapbox",
+    "photon",
+    "what3words",
+    "mapquest",
+    "mapzen",
+    "here"
+  ],
+  "author": "Per Liedman <per@liedman.net>",
+  "license": "BSD-2-Clause",
+  "dependencies": {
+    "leaflet": "~0.7.7"
+  },
+  "homepage": "http://www.liedman.net/leaflet-control-geocoder/",
+  "npmName": "leaflet-control-geocoder",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(*.+(png|gif))"
```

Comment:
because there are same `*.(png|gif)` files in `/dist/image`, you said we don't need to add the same files twice.
>Why do we need to add them twice? I didn't see a reason ...

so I use `!(*.+(png|gif))` to avoid adding them twice.